ClamXav/ClamXav.download.recipe

```diff
@@ -3,27 +3,26 @@
 <plist version="1.0">
 <dict>
 	<key>Comment</key>
-	<string>Created with Recipe Robot v1.0.3 (https://github.com/homebysix/recipe-robot)</string>
+	<string>Created with Recipe Robot v2.0.0 (https://github.com/homebysix/recipe-robot)</string>
 	<key>Description</key>
 	<string>Downloads the latest version of ClamXav.</string>
 	<key>Identifier</key>
 	<string>com.github.homebysix.download.ClamXav</string>
 	<key>Input</key>
+	<key>Input</key>
```

Comment:
This appears to be a duplicated key, and is preventing the plist from successfully parsing.

Reply:
Sorry, fixed it!